### PR TITLE
Terraform scenario upgrades for 0.14, small improvements

### PR DESCRIPTION
**Pull request by ritsok**
Most of the work in this PR is to upgrade to 0.14, though I did some opportunistic cleanup as well. The most substantial changes are to the build-modules scenario, which you can test out [here](https://www.katacoda.com/ritsok/scenarios/terraform-build-modules).

### CODE REVIEW

File: terraform-build-modules/index.json [hunk 1]
```diff
@@ -41,5 +42,5 @@
   "backend": {
     "imageid": "ubuntu"
   },
-  "private": true
+  "private": false
```
Comment (im2nguyen) [inline terraform-build-modules/index.json:45]: remember to toggle this back to private before you merge!

File: terraform-build-modules/step4.md [hunk 2]
```diff
@@ -4,41 +4,45 @@ module before it can be used. Both the `terraform get` and `terraform init`
...
-`terraform output website_bucket_name`{{execute}}
-
-Then, visit the website endpoint in a web browser, replacing `<BUCKET_NAME>` 
-with your bucket name. You should find the website contents.
+Visit the website endpoint below in a web browser. You should find the website
+contents.
 
-`[[HOST_SUBDOMAIN]]-4566-[[KATACODA_HOST]].environments.katacoda.com/${module.website_s3_bucket.name}/index.html`
\ No newline at end of file
+[[HOST_SUBDOMAIN]]-4566-[[KATACODA_HOST]].environments.katacoda.com/terraform-edu-modules/index.html
```
Comment (im2nguyen) [inline terraform-build-modules/step4.md:48]:
```suggestion
<pre>[[HOST_SUBDOMAIN]]-4566-[[KATACODA_HOST]].environments.katacoda.com/terraform-edu-modules/index.html</pre>
```

hrm, this didnt create a link for some reason (and didn't interpolate the content), can you try the above suggestion? this one is lifted directly from [katacoda example scenario](https://github.com/katacoda/scenario-examples/blob/main/placeholders/step2.md)

image

**Review by im2nguyen — APPROVED** [review]
Overall looks great! One small nit and another reminder!

File: terraform-build-modules/index.json [hunk 3]
```diff
@@ -41,5 +42,5 @@
   "backend": {
     "imageid": "ubuntu"
```
Comment (tr0njavolta) [inline terraform-build-modules/index.json:43]: The vanilla ubuntu image returns a Python 3.5 deprecation warning, but testing with `ubuntu:2004` did not. More to do with the katacoda image than the content though.

Screen Shot 2021-02-24 at 7 28 12 PM

File: terraform-build-modules/step2.md [hunk 4]
```diff
@@ -14,8 +14,6 @@ should look like the following.
 ├── modules
 │   └── aws-s3-static-website-bucket
 ├── outputs.tf
-├── terraform.tfstate
-├── terraform.tfstate.backup
 └── variables.tf
```
Comment (tr0njavolta) [inline terraform-build-modules/step2.md:17]: Add assets directory.

**Review by tr0njavolta — APPROVED** [review]
A couple of small nits but looks great!

**Review by judithpatudith — APPROVED** [review]
Ran into the same issue as Tu accessing the modules endpoint. Other than that LGTM

**Pull request merged**